core/src/services/fs/backend.rs

```diff
@@ -503,7 +503,7 @@ impl Accessor for FsBackend {
 
                 Ok(RpDelete::default())
             }
-            Err(err) if err.kind() == std::io::ErrorKind::NotFound => Ok(RpDelete::default()),
+            Err(err) if err.kind() == io::ErrorKind::NotFound => Ok(RpDelete::default()),
```

Comment:
I perfer to use `std::io::ErrorKind::NotFound` here. We can remove the use `std::io` instead.